src/components/study-container.js

```diff
@@ -316,16 +316,26 @@ export function StudyContainer({ view, onChangeTab }) {
                     )
                 );
             })
-            .catch((errorMessage) =>
-                displayErrorMessageWithSnackbar({
-                    errorMessage: errorMessage,
-                    enqueueSnackbar: enqueueSnackbar,
-                    headerMessage: {
-                        headerMessageId: 'NetworkModificationTreeLoadError',
-                        intlRef: intlRef,
-                    },
-                })
-            )
+            .catch((errorMessage) => {
+                if (errorMessage.status === 404)
+                    displayErrorMessageWithSnackbar({
+                        errorMessage: '',
+                        enqueueSnackbar: enqueueSnackbar,
+                        headerMessage: {
+                            headerMessageId: 'StudyAwfulStateRecreate',
```

Comment:
not a big fan of the naming "awful" but whatever

Reply:
Not very proud of, but it is internal. If you are inspired, feel free to propose another name ;-)

Reply:
StudyUnrecoverableStateRecreate could be a better property name

Reply:
+1